pkg/gen/input/ami/funcs.go

```diff
@@ -1,122 +1,67 @@
 package ami
```

Comment:
Only the scraping part is left in this file. It gets the values to populate template data.